Libraries/LogBox/LogBoxInspectorContainer.js

```diff
@@ -21,9 +21,10 @@ type Props = $ReadOnly<{|
 |}>;
 
 export class _LogBoxInspectorContainer extends React.Component<Props> {
+  const {width, height} = Dimensions.get('window');
```

Comment:
This line causes a syntax error here, but if placed below inside the top of the render() function, it works.